src/tasks/tasks.py

```diff
@@ -543,6 +546,101 @@ def get_metrics(self, reset=False):
                 reset))
         return collected_metrics
 
+@register_task('cola-pair-frozen', rel_path='CoLA')
```

Comment:
Sorry to be annoying about this, but could you create a new file in `src/tasks/`? I'd like to avoid throwing more tasks, esp. tasks that are somewhat project specific, into this file.

Reply:
It's okay, I agree with you.